titan/probabilities.py

```diff
@@ -5,28 +5,20 @@
 
 
 @utils.memo
-def adherence_prob(adherence: int) -> float:
+def adherent_prob(adherent: bool) -> float:
     """
     Mapping from HAART adherence levels to probabilities.
 
     args:
-        adherence: HAART adherence level
+        adherent: Whether an agent is HAART adherent
 
     returns:
         probability of agent transitioning from HIV+ to AIDS
```

Comment:
should this still be a prob now that it's just two levels? or maybe put it in the haart params?